metadata-ingestion/src/datahub/ingestion/source/salesforce.py

```diff
@@ -537,16 +537,21 @@ def get_profile_workunit(
 
     # Here field description is created from label, description and inlineHelpText
     def _get_field_description(self, field: dict, customField: dict) -> str:
-        desc = (
-            "\\" + field["Label"] if field["Label"].startswith("#") else field["Label"]
-        )
+        if field["Label"] is None:
+            desc = ""
+        else:
+            if field["Label"] is not None and field["Label"].startswith("#"):
```

Comment:
Why do you need this `field["Label"] is not None`? Should `elif` not take care of that already given the first condition? Also please use `elif` instead of this nesting.